.circleci/config.yml

```diff
@@ -69,4 +88,5 @@ workflows:
           - prepare
       - test_server:
           requires:
-          - prepare
\ No newline at end of file
+          - prepare
+      - main
```

Comment:
would it require anything beforehand for it to run?

Reply:
no, it uses a circleci base image having the necessary tools like `docker-compose` and directly builds the docker image